gulpfile.js

```diff
@@ -170,3 +169,7 @@ gulp.task('prefetch', function(cb) {
   }
   artifactType = "-bundle";
 });
+
+gulp.task('default', function() {

```

Comment:
But transpile (which is what the run task does) is meaningless w/o first doing a generate, right?